### PR TITLE
Feat/long running operations

### PR DESCRIPTION
**Pull request by deleon117**
- Changed return type of SimulationService rpc to google.longrunning.Operation
- Added metadata for long running operation
- Added googleapi protos

### CODE REVIEW

**Review by safeerehman — APPROVED** [review]

**Comment by pkrull-ansys** [comment]
@deleon117 I forgot, we also need an OperationMetadata message defined. The details can be found in https://tfs.ansys.com:8443/tfs/ANSYS_Development/Portfolio/_workitems/edit/1020417.

**Review by pkrull-ansys — CHANGES_REQUESTED** [review]

File: ansys/api/additive/v0/additive_operations.proto [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import "google/protobuf/timestamp.proto";
+
+// Metadata for a long running operation
+message OperationMetadata {
+    google.protobuf.Timestamp CreateTime = 1;
+    google.protobuf.Timestamp EndTime = 2;
+    string simulationId = 3;
+    string context = 4;
+    string message = 5;
+    float percentComplete = 6;
```
Comment (pkrull-ansys) [inline ansys/api/additive/v0/additive_operations.proto:18]:
```suggestion
message OperationMetadata {
    google.protobuf.Timestamp creation_time = 1;
    google.protobuf.Timestamp end_time= 2;
    string simulation_id = 3;
    string context = 4;
    string message = 5;
    float percent_complete = 6;
```

For better or worse, I used snake case when defining my messages previously. I'm just wanting to keep things consistent.

**Review by pkrull-ansys — APPROVED** [review]
Just needs one naming convention change.

File: ansys/api/additive/v0/additive_operations.proto (hunk 1 above)
Comment (deleon117) [reply ansys/api/additive/v0/additive_operations.proto]: Changed to snake case

**Pull request merged**